libsql-server/src/connection/config.rs

```diff
@@ -29,6 +33,8 @@ pub struct DatabaseConfig {
     pub is_shared_schema: bool,
     #[serde(default)]
     pub shared_schema_name: Option<NamespaceName>,
+    #[serde(default)]
+    pub durability_mode: Option<DurabilityMode>,
```

Comment:
Since we have an off mode, I think the option is not necessary. Let'a implement default for mode and use that directly